tests/kafkatest/services/console_consumer.py

```diff
@@ -107,7 +107,7 @@ def __init__(self, context, num_nodes, kafka, topic, new_consumer=False, message
             num_nodes:                  number of nodes to use (this should be 1)
             kafka:                      kafka service
             topic:                      consume from this topic
-            new_consumer:               use new Kafka consumer if True
+            new_consumer:               use new Kafka consumer if True. If not set, value is determined based on security_protocol
```

Comment:
Is this still true? I don't see logic any more that sets this based on security_protocol